dependencies.gradle

```diff
@@ -17,11 +17,13 @@ dependencies {
 
     compile("com.github.GTNewHorizons:TCNodeTracker:latest")
 
+    compile("com.github.GTNewHorizons:IFU:b33994723f:dev")
```

Comment:
for now it targets IFU from my [PR branch](https://github.com/GTNewHorizons/IFU/pull/5) (latest commit), I guess it's fine from build system point of view (jitpack is immutable), but it would be better to target thing that is on the master so we are sure it is in the pack

Reply:
I would suggest having concrete versions, so it's stable in case you want to checkout older commits. Also of something is broken in dependency you might have unexpected failures

Reply:
This of course has a downside of needing to update things once they moved in the pack, but since there will be changes in the code - you'll need to make a PR anyway